leveler/commands/lvladmin/settings.py

```diff
@@ -61,6 +62,13 @@ async def overview(self, ctx):
         )
         await ctx.send(embed=em)
 
+    @checks.is_owner()
+    @lvladmin.command()
+    async def resetrep(self, ctx):
+        """Resets all reputation points from MonogoDB (Bot Owner Only)"""
```

Comment:
Others commands docstrings doesn't have this notices
```suggestion
        """Resets all reputation points from MonogoDB"""
```